plugin/src/main/kotlin/nl/avisi/anowalke/service/Sanitiser.kt

```diff
@@ -0,0 +1,37 @@
+package nl.avisi.anowalke.service
+
+import com.intellij.openapi.diagnostic.Logger
+import nl.avisi.anowalke.dto.ExpressionsDto
+
+class Sanitiser {
+
+    private val restService = RestService()
+
+    private var camelCaseTitleCaseRegex = "(?<!(^|[A-Z]))(?=[A-Z])|(?<!^)(?=[A-Z][a-z])".toRegex()
+
+    private val log = Logger.getInstance(this::class.java.name)
+
+    fun sanitise(expressions: List<String>): List<String> =
+        // Stap 1: Remove correct calls
+
+        // Kan hier niet want dit gebeurt recursief?
+
+
+        // Stap 2: Split expressies aan de hand van regex
+        // Stap 3: Transform to lowercase, misschien al bij stap 2 erin
+        // Stap 4: Remove duplicates
+        // Stap 5: Remove irrelevant words -> pas in de api uitvoeren? Op basis van database? Eerste scan op basis van stopwoordenlijst?
+        expressions.split().toLowercase().distinct().translate().also { log.warn("Initiate sanitise function.") }
```

Comment:
Beter leesbaar als je elke nieuwe functie op een nieuwe regel start in deze chain